Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -594,48 +594,48 @@ Don't have a 4chan Pass?<br>
-    <string name="captcha_layout_v2_verification_already_in_progress">Verification is already in progress</string>
-    <string name="captcha_layout_v2_captcha_request_is_already_in_progress">Captcha request is already in progress</string>
-    <string name="captcha_layout_v2_you_are_requesting_captcha_too_fast">You are requesting captcha too fast</string>
+    <string name="captcha_layout_v2_you_have_to_select_at_least_one_image">Select at least one image (if you see a CAPTCHA that does not have any matching images you can file a bug and use the old one)</string>
+    <string name="captcha_layout_v2_verification_already_in_progress">Verification already in progress…</string>
+    <string name="captcha_layout_v2_captcha_request_is_already_in_progress">CAPTCHA request already in progress…</string>
+    <string name="captcha_layout_v2_you_are_requesting_captcha_too_fast">You requested too many CAPTCHAs in a short timespan.</string>
     <string name="image_url_copied_to_clipboard">Image URL copied to clipboard</string>
     <string name="delete_site_dialog_title">Delete site?</string>
-    <string name="delete_site_dialog_message">Are you sure you want to delete "%1$s"? This action will delete the site and everything that is associated with it. The app will be restarted. Do you wish to proceed?</string>
+    <string name="delete_site_dialog_message">Really delete "%1$s" (meaning the site and everything associated with it and then restart the app)?</string>
 
     <string name="could_not_remove_site_error_message">Could not remove site %1$s (error message: %2$s)</string>
 
-    <string name="reply_comment_empty">Comment body is empty.</string>
+    <string name="reply_comment_empty">Comment body empty.</string>
     <string name="filter_removed_undo">Removed filter %1$s</string>
     <string name="image_url_get_attempt">Getting image from clipboard URL…</string>
-    <string name="image_url_get_failed">Failed to get image from clipboard; %1$s</string>
+    <string name="image_url_get_failed">Could not fetch image from clipboard because %1$s</string>
     <string name="image_url_get_success">Got image from clipboard</string>
     <string name="setting_shift_post">Shift post format</string>
-    <string name="setting_shift_post_description">Shift the comment of long posts to start below the thumbnail</string>
+    <string name="setting_shift_post_description">Move the comment of long posts to start below the thumbnail.</string>
     <string name="setting_remove_watched">Remove watched threads from the catalog</string>
 
-    <string name="setting_enable_emoji">Enable emoji</string>
-    <string name="setting_enable_emoji_description">Enable emoji posting and parsing</string>
+    <string name="setting_enable_emoji">Emoji</string>
+    <string name="setting_enable_emoji_description">Allows seeing and pasting smiley graphics.</string>
```

Comment:
Old was better

Reply:
It fails if the user doesn't know what those are.